src/plugins/inspector/public/views/requests/components/details/req_code_viewer.tsx

```diff
@@ -88,6 +106,18 @@ export const RequestCodeViewer = ({ indexPattern, json }: RequestCodeViewerProps
               {openInConsoleLabel}
             </EuiButtonEmpty>
           )}
+          {shouldShowsearchProfilerLink && (
```

Comment:
Would it be possible to add a slightly larger horizontal space between the buttons? At the moment it is a bit visually tricky to group the icon and the text at a glance. If this messes to much with limited horizontal space, please ignore!

Reply:
Updated the layout to use flex box:

<img width="841" alt="Screen Shot 2022-03-21 at 1 29 03 PM" src="https://user-images.githubusercontent.com/5226211/159330635-433f7007-9aa6-4aab-b62b-e2982e6928d1.png">